media/css/sandstone/sandstone-resp.less

```diff
@@ -899,7 +899,7 @@ nav.menu-bar {
     }
 
     input[type=email] {
-        width: 80%;
+        width: 92%;
```

Comment:
I think you can use the `.border-box;` mixin here (which sets `box-sizing: border-box;`, the mixin just includes the vendor prefixes for browsers that need them) and set this width to 100%.